config/node.options

```diff
@@ -0,0 +1,8 @@
+## Node command line options
+## See `node --help` and `node --v8-options` for available options
```

Comment:
Would probably be good to note that it's one option per line.

Reply:
👍